packages/aragon-wrapper/src/index.js

```diff
@@ -435,8 +435,8 @@ export default class Aragon {
     // These may modify the implementation addresses of the proxies (modifying their behaviour), so
     // we invalidate any caching we've done
     const updatedApps$ = this.kernelProxy
-      // Override events subscription with empty options to subscribe from latest block
-      .events('SetApp', {})
+      // Only need to subscribe from latest block
+      .events('SetApp', { fromBlock: 'latest' })
```

Comment:
From [event](https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#contract-events) the docs say that `fromBlock` only accepts a only a `number`. In other web3 methods it accepts a `string`. 
Have you tested this?

Reply:
Not explicitly, but we should! I remember the web3.js code itself checking for 'latest'.

Reply:
It turns out that in web3.js 2.x, [`'latest'` is checked explicitly](https://github.com/ethereum/web3.js/blob/2.x/packages/web3-core-subscriptions/src/subscriptions/eth/LogSubscription.js#L53) but in web3.js 1.x, ['latest' implicitly skips fetching the past logs since it's not a finite number](https://github.com/ethereum/web3.js/blob/1.x/packages/web3-core-subscriptions/src/subscription.js).

The [EIP specifying the RPC calls does note 'latest' as a possible value, amongst others](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md#eth_getlogs), and the [wiki's json-rpc entry on `getLogs`](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getlogs) notes that 'latest' should be the default value for `fromBlock` (and indeed, web3.js is programmed with that assumption).

Even though web3.js 1.x doesn't handle `'latest'` in the _best_ way, given that it is still _working_ and `'latest'` is a possible value for the JSON-RPC API's `fromBlock`, I'm inclined to keep this as it's explicit and any potential regression would squarely be a bug in web3.js.